src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionTimeoutErrorInternal.cs

```diff
@@ -58,23 +60,21 @@ internal long GetMilliSecondDuration()
 
     internal class SqlConnectionTimeoutErrorInternal
     {
-        private SqlConnectionTimeoutPhaseDuration[] _phaseDurations = null;
-        private SqlConnectionTimeoutPhaseDuration[] _originalPhaseDurations = null;
-
-        private SqlConnectionTimeoutErrorPhase _currentPhase = SqlConnectionTimeoutErrorPhase.Undefined;
-        private SqlConnectionInternalSourceType _currentSourceType = SqlConnectionInternalSourceType.Principle;
-        private bool _isFailoverScenario = false;
+        private SqlConnectionTimeoutPhaseDuration[] _phaseDurations;
+        private SqlConnectionTimeoutPhaseDuration[] _originalPhaseDurations;
+        private SqlConnectionTimeoutErrorPhase _currentPhase;
+        private SqlConnectionInternalSourceType _currentSourceType;
+        private bool _isFailoverScenario;
```

Comment:
Any specific reason to remove these defaults?

Reply:
They were the default/zero values for each type and the language requires that the be zero value on first access. Assigning to them at declaration point forces the creation of a ctor by the compiler. It was just unneeded noise.